JS/Lesson34/goksel/recipe-app/src/App.css

```diff
@@ -1,24 +1,19 @@
-.grey {
-  background-color: grey;
-} 
-
-.yellow {
-  background-color: yellow;
-}
-
-.white {
-  background-color: white;
+.appRouter {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
 }
 
-.recipes__list {
-  list-style-type: none;
-  padding-left: 0;
+.newRecipeBtn {
```

Comment:
it would be better to use BEM naming convention. Using camel case for naming classes is not a good practice.

Reply:
I removed that css property, i figured that ı am not using it anymore